chain/syncer_integration_test.go

```diff
@@ -112,267 +108,44 @@ func TestLoadFork(t *testing.T) {
 	assert.NoError(t, offlineSyncer.HandleNewTipSet(ctx, types.NewChainInfo("", left.Key(), heightFromTip(t, left)), true))
 }
 
-// Syncer handles PowerTableView weight comparisons.
-// Current issue: when creating miner mining with addr0, addr0's storage head isn't found in the blockstore
-// and I can't figure out why because we pass in the correct blockstore to createStorageMinerWithpower.
-func TestTipSetWeightDeep(t *testing.T) {
-	// This test takes many seconds, the bottleneck is gengen.
-	tf.IntegrationTest(t)
+// Power table weight comparisons impact syncer's selection.
+// One fork has more blocks but less total power.
+// Verify that the heavier fork is the one with more power.
+func TestSyncerWeighsPower(t *testing.T) {
+	t.Skip("fill this out when making weight upgrade")
```

Comment:
It would make me more comfortable leaving this in the code if you either added a TODO with an issue to point to the future work or include the issue # in the skip message (this applies to the next test too).

Reply:
Woops I meant to delete this.  Anorth requested a rebase and this tagged along unintended.